source/tyr/tyr/resources.py

```diff
@@ -1691,43 +1694,82 @@ def check_db():
         return None
 
 
+@marshal_with(job_fields)
+def check_cities_job():
+    """
+    Check status of cities job in Tyr db
+    :return: the latest cities job
+    """
+    return (
+        models.Job.query.join(models.DataSet)
+        .filter(models.DataSet.type == 'cities')
+        .order_by(models.Job.created_at.desc())
+        .first()
+    )
+
+
 class CitiesStatus(flask_restful.Resource):
     def get(self):
+        response = {}
         if not current_app.config['CITIES_DATABASE_URI']:
             return {'message': 'cities db not configured'}, 404
-        msg = check_db()
-        if msg:
-            return {'message': 'cities db alembic version = {}'.format(msg)}, 200
+        cities_version = check_cities_db()
+        if cities_version:
+            response['cities db version'] = '{}'.format(cities_version)
         else:
             return {'message': 'cities db not reachable'}, 404
 
+        cities_job = check_cities_job()
+        if cities_job and 'instance' in cities_job:
+            # No instance associated to 'cities' job, remove the item
```

Comment:
I don't understand this.
I'm not used to tyr though, but it may be worth explaining a bit more (or rewrite comment?) if I'm not the only one.

Reply:
Let's wait to see if you're not the only one...

Reply:
The explanation on this one is rather long and not really useful for understanding the cities status.